decidim-core/app/types/decidim/core/user_entity_input_filter.rb

```diff
@@ -46,6 +46,16 @@ class UserEntityInputFilter < BaseInputFilter
                    op_name.or(op_nick)
                  end
                end
+      argument :exclusion,
+               type: String,
+               description: "Excludes users contained in given ids",
```

Comment:
```suggestion
               description: "Excludes users contained in given ids. Valid values are one or more IDs (passed as an arrays)",
```
This is used in the documentation, so let's try to be as clear as possible